internal/publish/model/exposure_model.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/google/exposure-notifications-server/internal/verification"
 	"github.com/google/exposure-notifications-server/pkg/base64util"
 
-	verifyapi "github.com/google/exposure-notifications-server/pkg/api/v1alpha1"
+	v1 "github.com/google/exposure-notifications-server/pkg/api/v1"
```

Comment:
I think it was a little more readable when this was `verifyapi`. `v1` is very general.